sdk/react/widgets/videoWaterMark.js

```diff
@@ -0,0 +1,89 @@
+var React = require('react-native');
+var {
+  Image,
+  Text,
+  StyleSheet,
+  Animated,
+  View,
+} = React;
+var ResponsiveDesignManager = require('../responsiveDesignManager');
+var Constants = require('../constants');
+var styles = require('../utils').getStyles(require('./style/videoWaterMarkStyles.json'));
+var {
+  BUTTON_NAMES,
+  IMG_URLS,
+  UI_SIZES
+} = Constants;
+
+
+
+var logo = React.createClass({
+  propTypes: {
+    frameWidth: React.PropTypes.number,
+    frameHeight: React.PropTypes.number,
+    buttonWidth: React.PropTypes.number,
+    buttonHeight: React.PropTypes.number,
+    platform: React.PropTypes.string,
```

Comment:
I didn't find any line that uses the platform in this file. It's okay if we remove it then.